skops/card/tests/test_card.py

```diff
@@ -17,28 +17,16 @@ def fit_model():
 
 def generate_card():
     model = fit_model()
-    card_data = CardData(library_name="sklearn")
-
-    model_card = create_model_card(
-        model,
-        card_data,
-        template_path="skops/card/default_template.md",
-        model_description="sklearn FTW",
-    )
+    model_card = Card(model)
     return model_card
 
 
-def test_write_model_card():
+def test_save_model_card():
     with tempfile.TemporaryDirectory(prefix="skops-test") as dir_path:
```

Comment:
You could use a fixture to simplify the tests on the tempdirectory like here:

https://github.com/skops-dev/skops/blob/64366d04a6f49777495b7cce1f13feede8f8b84e/skops/hub_utils/tests/test_hf_hub.py#L25-L28

Reply:
This is not fixed yet @merveenoyan .

Reply:
you can also have a look at how I'm changing these tests here: https://github.com/skops-dev/skops/pull/45/files#diff-0d678309e10b981c043c9faa2f9b825e9e13fafdce4eed8cc371b8d61874fd26